common/app/redux/reducer.js

```diff
@@ -11,14 +11,15 @@ const initialState = {
   navHeight: 0,
   isMainChatOpen: false,
   isHelpChatOpen: false,
-  theme: 'default'
+  theme: 'default',
+  block: ' '
 };
 
 export default handleActions(
   {
     [types.updateTitle]: (state, { payload = 'Learn To Code' }) => ({
       ...state,
-      title: payload + ' | Free Code Camp'
+      title: `${ state.block || 'Free Code Camp' }: ${payload} | Free Code Camp`
```

Comment:
Should we not have it like:
`title: `${ state.block || 'Challenge' }: ${payload} | Free Code Camp``